src/modules/commentModal.js

```diff
@@ -0,0 +1,39 @@
+// eslint-disable-next-line import/no-unresolved
```

Comment:
- [ ] Kindly fix those errors flagged by Linters. Linters are great tools :hammer: that enable you to catch programming bugs. Avoid disabling them, please.